.github/workflows/build-and-test-orchard-core.yml

```diff
@@ -89,12 +89,12 @@ jobs:
         - ${{ inputs.timeout-minutes }}
     timeout-minutes: ${{ matrix.timeout-minutes }}
     steps:
-    - uses: actions/checkout@v3
```

Comment:
Are all these warnings coming from Test Reports? https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/3362208732 Please run a build with just that step removed to see if all vanish.

Reply:
Yes, you can see the results on [this run](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/3364050166/jobs/5577951452).

Reply:
This still has many warnings. The warnings show up on the Summary page, not under each job, like here at the bottom: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/3364050166

Reply:
They are from the non-nuget builds, because they are using the dev branch of the workflow, as you requested [here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/241#discussion_r1009726869). In every build the Tests step generates 6 warnings, that's why we had 24, and on this run we only have 12.

Reply:
OK, then please demonstrate this with using the issue branch for just this build.

Reply:
[Here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/3365284817) is the run.